auditlog/models/log.py

```diff
@@ -14,7 +14,7 @@ class AuditlogLog(models.Model):
     res_id = fields.Integer("Resource ID")
     user_id = fields.Many2one(
         'res.users', string="User")
-    method = fields.Char("Method", size=64)
+    method = fields.Char(size=64)
```

Comment:
Maybe we could also get rid of the `size`. Any good reason to keep it?

Reply:
No good reason to me.